ktor-bom/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'io.spring.dependency-management'
+}
+
+apply from: rootProject.file('gradle/publish.gradle')
+
+def name = project.name
+
+dependencyManagement {
+    dependencies {
+        rootProject.subprojects.each {
+            if (it.plugins.hasPlugin("maven-publish") && it.name != name) {
+                it.publishing.publications.all {
+                    if (it.artifactId.endsWith("-metadata")) return
+                    if (it.artifactId.endsWith("-kotlinMultiplatform")) return
+                    dependency(group: it.groupId, name: it.artifactId, version: it.version)
+                }
+            }
+        }
+    }
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
```

Comment:
What does it produce? What `packaging` it will produce? artifact id and so on? maven central have quite strict requirements for pom files

Reply:
It produce standard artifact like all others: io.ktor:ktor-bom, packaging is pom, all pom metadata handled by `gradle/publish.gradle` just like all others artifacts in the repo.
All maven central restrictions are satisfied.